source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1395,6 +1444,58 @@ message TransformationMergeBlocks {
 
 }
 
+message TransformationMergeFunctionReturns {
+
+  // A transformation that modifies a function so that it does not return early,
+  // so it only has one return statement (ignoring unreachable blocks).
+  //
+  // The function is enclosed inside an outer loop, that is only executed once,
+  // and whose merge block is the new return block of the function.
+  //
+  // Each return instruction is replaced by:
+  //     OpBranch %innermost_loop_merge
+  // where %innermost_loop_merge is the innermost loop containing such return.
+  //
+  // Each merge block containing return instructions is changed so that it
+  // branches to the merge block of the loop containing it, as explained in
+  // the comments to the ReturnMergingInfo message.
+  //
```

Comment:
I found this comment misleading.  Do you really mean "each merge block containing return instructions" - in that the merge block itself has to contain a merge instruction?  Or do you mean "Each merge block whose associated loop originally contained return instructions"?

Reply:
The first one. I changed the comment to make it clearer